### PR TITLE
feat / quick start - CLI entry point

### PR DESCRIPTION
**Pull request by yzhang1994**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:
- Added a new entry point for quick start  
```
bin/hummingbot_quickstart.py \
--strategy ${STRATEGY} \
--config-file-path ${CONFIG_FILE_PATH} \
--wallet ${WALLET} \
--wallet_password ${WALLET_PASSWORD} 
```
- Modified docker file so that a container can be automatically restarted
```
docker run -it \
-e STRATEGY=${STRATEGY} \
-e CONFIG_FILE_PATH=${CONFIG_FILE_PATH} \
--mount "type=bind,source=$(pwd)/$FOLDER/hummingbot_conf,destination=/conf/" \
--mount "type=bind,source=$(pwd)/$FOLDER/hummingbot_conf,destination=/logs/" \
coinalpha/hummingbot:${TAG}
```

### CODE REVIEW

**Comment by dennisocana** [comment]
`--wallet_password` not working, will return a message:
```
Bot config incomplete: unrecognized arguments: --wallet_password [**DELETED**]. Starting normally...
```
Use `--wallet-password` instead for bot to auto-start.

Including the folder for `--config-file-path` (e.g. `conf/pure_market_making_strategy_0.yml`) isn't being accepted. Would work by omitting the folder and enter `pure_market_making_strategy_0.yml` only.

Suggestion: change to `--config-file-name`

**Comment by martinkou** [comment]
Concur with the `config-file-name` suggestion. Please change the argument name.

File: bin/hummingbot_quickstart.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+class CmdlineParser(ThrowingArgumentParser):
+    def __init__(self):
+        super().__init__()
+        self.add_argument("--strategy", "-s",
+                          type=str,
+                          choices=STRATEGIES,
+                          required=True,
+                          help="Choose the strategy you would like to run.")
+        self.add_argument("--config-file-path", "-f",
```
Comment (martinkou) [inline bin/hummingbot_quickstart.py]: Please change to `config-file-name`

**Comment by dennisocana** [comment]
Performed testing on modified Dockerfile with pure-mm + ddex

- Including the folder for `CONF_FILE_PATH` also doesn't work (e.g. conf/conf_pure_market_making_strategy_0.yml)

test5

- Omitting the folder is accepted and use conf_pure_market_making_strategy_0.yml instead

test4

**Comment by yzhang1994** [comment]
@dennisocana @martinkou Changed the argument name.

**Pull request merged**